java-apikeys/samples/install-without-bom/pom.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
This file shouldn't be included along with other poms in `samples/`

Reply:
True.